classes/invitation/steps/Step.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace PKP\invitation\steps;
+
+use stdClass;
+
+class Step
+{
+    public string $id;
+    public string $type;
+    public string $name;
+    public string $description;
+    public string $stepLabel;
+    public string $nextButtonLabel;
+    public array $sections = [];
+
+    /**
+     * @param string $id A unique id for this step
+     * @param string $name The name of this step. Shown to the user.
+     * @param string $description A description of this step. Shown to the user.
```

Comment:
@param string $description A description of this step , shown to the user.

Reply:
done